cart/views.py

```diff
@@ -0,0 +1,99 @@
+from django.http import HttpResponse
+from django.shortcuts import render, redirect, reverse
+from django.contrib import messages
+
+from products.models import Product
+
+# Create your views here.
+
+def view_cart(request):
+    """A view that renders the cart contents page"""
+    return render(request, "cart/cart.html")
+    
+    
+def add_to_cart(request, item_id):
+    """Add a quantity of the specified product to the cart"""
+    
+    quantity = int(request.POST.get('quantity'))
+    size = None
+    if 'product_size' in request.POST:
+        size = request.POST['product_size']
+    cart = request.session.get('cart', {})
+    product = Product.objects.get(id=item_id)
```

Comment:
I'd possibly change this product fetching line to use `get_or_404` and be the very first line of the function

Reply:
Yep - updated!